internal/rbd/controllerserver.go

```diff
@@ -252,6 +252,9 @@ func buildCreateVolumeResponse(req *csi.CreateVolumeRequest, rbdVol *rbdVolume)
 		}
 	}
 
+	// remove topologyConstrainedPools param as it is not required
```

Comment:
It feels cleaner to do this at the place where other keys to `volumeContext` are added or removed. Maybe directly below `k8s.RemoveCSIPrefixedParameters()` ?

Reply:
yea, that will be good.
How about introducing a function called `RemoveVolumeContextParameters`? This function could serve the specific purpose of removing parameters from the volume context."

Reply:
Sure, or make that a `volumeContext := rbdVol.getVolumeContext(req.GetParameters())` function.